app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt

```diff
@@ -54,6 +70,12 @@ class SettingsFragment : PreferenceFragmentCompat(), SettingsView {
             it.summary = "${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})"
         }
 
+        findPreference<SwitchPreference>("fullscreen")?.onPreferenceChangeListener =
+            onChangeSwitchFullScreen
+
+        findPreference<SwitchPreference>("hide_nav")?.onPreferenceChangeListener =
```

Comment:
Let's not add this option. That's feature creep. You go full screen or not.

Reply:
We need to be cautious with the number of options that we show. If it's too many, none will be used. We need to aim for what is right for 80% of the users.